lib/back.js

```diff
@@ -335,4 +335,4 @@ Back.fixtures = null;
 Back.currentMode = null;
 Back.setMode(process.env.NOCK_BACK_MODE || 'dryrun');
 
-module.exports = exports = Back;
+module.exports = Back;
```

Comment:
[Assigning to `exports` does not do anything.](https://lgtm.com/rules/10030091/)